### PR TITLE
[All] (Centralized deployment) Update to current flow

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[369a491](https://github.com/OfficeDev/office-js-docs-pr/commits/369a4919f4bc602c98e40e9b8e2e86dfbf251d2b)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1) | :x:Error | | [Details](#user-content-402d9d56-71db7a04)
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/.openpublishing.publish.config.json) | :x:Error | | [Details](#user-content-d902ab53-71db7a04)
[]() | :x:Error | | [Details](#user-content-71db7a04)

<a id="402d9d56-71db7a04"></a>
### [_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1)
 - **Line 82**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Exit code: -1073741502`````
 - **Line 217**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="d902ab53-71db7a04"></a>
### [.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/.openpublishing.publish.config.json)
 - **Line 80**: **[Error]** `````Failed to execute customized script at step docset_prebuild in docset office-js-docs-pr. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins for docset office-js-docs-pr step docset_prebuild or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="71db7a04"></a>
###
 - **Line 138**: **[Error]** `````W:\2pni-s\.optemp\packages\opbuild.scripts.1.84.4\tools\opbuild\docset.prebuild.ps1 failed to execute`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002131654452988-1593%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=4KCKGhdozNi5lhlN0cQAd02Ag0rVqGPQXX651TaH%2FRo%3D&st=2020-02-13T16%3A50%3A34Z&se=2020-03-15T16%3A55%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/publish/centralized-deployment.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Office 365 admin center makes it easy for an administrator to deploy Office
 The Office 365 admin center currently supports the following scenarios:
 
 - Centralized Deployment of new and updated add-ins to individuals, groups, or an organization.
-- Deployment to multiple platforms, including Windows, Mac, and on the web.
+- Deployment to multiple platforms, including Windows, Mac, mobile, and on the web.
```
Comment (bhavinkap) [inline docs/publish/centralized-deployment.md]: Should we call out Tabs like iPad separately. Or can we look to the usage of term "mobile devices" to encompass Tablets

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6aa181a](https://github.com/OfficeDev/office-js-docs-pr/commits/6aa181a5498f4cadd2c7e63a851bd02308f6fb48)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/manage-users-deployment-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/manage-users-deployment-admin-center.png) | :white_check_mark:Succeeded | |
[docs/images/power-bi-tiles-edit.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/power-bi-tiles-edit.png) | :white_check_mark:Succeeded | |
[docs/images/power-bi-tiles.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/power-bi-tiles.png) | :white_check_mark:Succeeded | |
[docs/images/selected-add-in-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/selected-add-in-admin-center.png) | :white_check_mark:Succeeded | |
[docs/images/update-add-in-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/update-add-in-admin-center.png) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002141714472452-1593%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=k6kMIRd3KbOMyYhii0r%2BCP%2BFTbkqbaHTI2UxS%2F%2BwZqo%3D&st=2020-02-14T17%3A11%3A36Z&se=2020-03-16T17%3A16%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f299ece](https://github.com/OfficeDev/office-js-docs-pr/commits/f299ecee4d09c32b315861f5af77a9cf3fe4ee2c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/manage-users-deployment-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/manage-users-deployment-admin-center.png) | :white_check_mark:Succeeded | |
[docs/images/power-bi-tiles-edit.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/power-bi-tiles-edit.png) | :white_check_mark:Succeeded | |
[docs/images/power-bi-tiles.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/power-bi-tiles.png) | :white_check_mark:Succeeded | |
[docs/images/selected-add-in-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/selected-add-in-admin-center.png) | :white_check_mark:Succeeded | |
[docs/images/update-add-in-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/update-add-in-admin-center.png) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002181659173530-1593%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CtDIw5VKL5kaLWOf6P3cV7vtafmCV2MY5aCpifjnXto%3D&st=2020-02-18T16%3A55%3A25Z&se=2020-03-20T17%3A00%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a62141e](https://github.com/OfficeDev/office-js-docs-pr/commits/a62141e0edee019cc1251375189fe1da13385cbe)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/manage-users-deployment-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/manage-users-deployment-admin-center.png) | :white_check_mark:Succeeded | |
[docs/images/power-bi-tiles-edit.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/power-bi-tiles-edit.png) | :white_check_mark:Succeeded | |
[docs/images/power-bi-tiles.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/power-bi-tiles.png) | :white_check_mark:Succeeded | |
[docs/images/selected-add-in-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/selected-add-in-admin-center.png) | :white_check_mark:Succeeded | |
[docs/images/update-add-in-admin-center.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/images/update-add-in-admin-center.png) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-cd/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002181718301345-1593%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7W8vLniLNpfBLK3UIxmnqmcGmQ%2BoxmSjXctm0D%2FCi8I%3D&st=2020-02-18T17%3A15%3A58Z&se=2020-03-20T17%3A20%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**